contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2638,20 +2638,6 @@ $function$
 ;
 GRANT EXECUTE ON FUNCTION sys.schema_name() TO PUBLIC;
 
-CREATE OR REPLACE FUNCTION sys.schema_id()
```

Comment:
just wondering, is there any specific reason to add the function definition in `import_export_compatibility.sql` instead of here

Reply:
Not sure about that. The previous declaration of the C-type function was in this file, so I did not change.